src/shared/ui/providers/query.tsx

```diff
@@ -1,48 +1,13 @@
-import {
-  MutationCache,
-  QueryCache,
-  QueryClient,
-  QueryClientProvider,
-} from '@tanstack/react-query';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import { useMemo } from 'react';
 
-import { useObservabilityScope } from 'shared/observability';
-
 interface Properties {
   children: React.ReactNode;
 }
 
 export const QueryProvider = ({ children }: Properties) => {
-  const observabilityScope = useObservabilityScope();
-
   const queryClient = useMemo(() => {
     return new QueryClient({
-      mutationCache: new MutationCache({
-        onError: (error, _variables, _context, mutation) => {
-          observabilityScope.setContext('mutation', {
-            mutationId: mutation.mutationId,
-            variables: mutation.state.variables,
-          });
-          if (mutation.options.mutationKey) {
-            observabilityScope.setFingerprint(
-              // Duplicate to prevent modification
-              [...mutation.options.mutationKey] as string[],
-            );
-          }
-          observabilityScope.captureException(error);
-        },
-      }),
-      queryCache: new QueryCache({
-        onError: (error, query) => {
-          observabilityScope.setContext('query', {
-            queryHash: query.queryHash,
-          });
-          observabilityScope.setFingerprint([
-            query.queryHash.replaceAll(/\d/g, '0'),
-          ]);
-          observabilityScope.captureException(error);
-        },
-      }),
```

Comment:
this was doubling the errors as api calls errors are already reported in command handlers, trade-off of removing this is that we won't catch errors from metamask but as discussed on Discord we will try to incrementally adopt on errors from metamask in order not to get blocked again